kubernetes/annotations_and_labels.md

```diff
@@ -19,14 +19,23 @@
 ## Common Labels (for guest cluster services)
 
 - `app` - value should contain the name of the application. Should be applied
-  to every Kubernetes resource associated with an application. This should 
+  to every Kubernetes resource associated with an application. This should
   also be used as replicas selector. E.g. `app=nginx-ingress-controller`.
   Exceptions can be made to accomodate for adherence to existing selectors
   upstream.
 - `giantswarm.io/service-type` - value should be either `system` for core system
   services (i.e. K8s components) or `managed` for Giant Swarm managed services
   (i.e. networking, DNS, monitoring, ingress controller, etc.). Latter would be
   managed by `chart-operator`.
+- `release.giantswarm.io/version` - value should be the used Giant Swarm release
```

Comment:
I think a separate heading for node labels would be good.

Reply:
Done.